### ISSUE TITLE
why the result seems not good

### ISSUE THREAD

**Issue by IVSUN** [wentaoyuan/deepgmr#4]
Hi，I‘ve read this paper. And I just want to say the idea is really brilliant！
Then I try to run this code ，however I have got a distinctive result.
here is the picture of the training process I run:
![f5a8cea8dafb7b72985f768e764eb1f](https://user-images.githubusercontent.com/35678005/98525944-23804a80-22b4-11eb-8485-d39435942174.jpg)
In your paper ,you said the epoch is 100 and I trained for more than a  half (50-60 epoch), but the rotation error and the translation error remained at a high level.
just like this:
Epoch 060 Step 246/276 Train: batch time 0.15, data time 0.1, loss 0.4664, rotation error 82.85, translation error 0.2303, RSME 0.4916

So should I  increase the training epoch? Or I do somthing wrong?

Besides,when I download the pretrained model and use the test data to test the model,  it report an error:
that seems the checkpoints is (64,80,1)  but the test data is (64,3,1)

It would be great If you could give some directions.
Thanks for your reading.

**Comment by wsunid** [comment]
Hi, I have the same issue here. Looking forward to hearing more.

Thanks!

**Comment by wentaoyuan** [comment]
You need to turn on the RRI feature with `--use_rri` flag for best results. Same goes for using the pretrained models, which all use RRI feature instead of raw XYZ as inputs.

**Comment by wsunid** [comment]
Hi Wentao,

Thanks much for the quick reply. 

Actually, I am trying to reproduce your experiments for the ablation study on the input feature (raw xyz vs the RRI). I would like to see the performance of the raw xyz. Is the performance of raw xyz that bad? Or do we need to change some other parameters (e.g., learning rate) in order to make the raw xyz work?

Thanks again!

Best regards,
Weiwei.

**Comment by wentaoyuan** [comment]
There was a bug in the model using XYZ. It computes the point cloud center using the wrong dimension. It should be fixed now. Please try again with the latest version of the code.
